App/View Controllers/Posts/PostsPageViewController.swift

```diff
@@ -531,7 +531,9 @@ final class PostsPageViewController: ViewController {
     }()
     
     
-    private lazy var actionsItem = UIBarButtonItem(title: "Menu", image: UIImage(named: "steamed-ham"), primaryAction: nil, menu: threadActionsMenu)
+    private func actionsItem() -> UIBarButtonItem {
```

Comment:
But this change does more, specifically dropping the `lazy` means the bar button item gets recreated each time.

(I'm happy with the change, this is just in case you were curious :)